h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -4357,3 +4357,9 @@ def assertCoefEqual(regCoeff, coeff, coeffClassSet, tol=1e-6):
         diff = abs(val1-val2)
         print("val1: {0}, val2: {1}, tol: {2}".format(val1, val2, tol))
         assert diff < tol, "diff {0} exceeds tolerance {1}.".format(diff, tol)
+
+def almost_equal(x, y, tol=1e-6):
```

Comment:
Just name `is_equal` please.

Reply:
Equals is probably better.

Reply:
Also, limit the arg type if possible in python.

Reply:
fixed